workitem/workitem_repository_blackbox_test.go

```diff
@@ -62,7 +62,7 @@ func (s *workItemRepoBlackBoxTest) TestFailDeleteZeroID() {
 		map[string]interface{}{
 			workitem.SystemTitle: "Title",
 			workitem.SystemState: workitem.SystemStateNew,
-		}, "xx")
+		}, uuid.NewV4())
```

Comment:
Previously all these tests worked with WI with the same creator "xx". Now it's generates a new UUID for every WI. Let's do both. Re-use the same UUID for some test WIs and re-generate a new UUID for others in tests where we create a few WIs. Just to make sure the DB doesn't care if the creator is not unique.